plugins/mcreator-localization/help/default/entity/condition_solid_bounding_box.md

```diff
@@ -0,0 +1,2 @@
+While the returned value is true, the entity's bounding box is solid.

```

Comment:
> While the returned value is true, the entity's bounding box is solid, meaning the entity can push it.

This may be a bit better wording for this?

Reply:
It can't be pushed while it is solid if not hitting it

Reply:
Then explain this, it is hard for a user to understand what solid means in terms of physics, where tooltips are there to explain what happens when parameter is enabled

Reply:
would "While the returned value is true, the entity's bounding box can be collided with like a block" be a clearer explanation?

Reply:
For reference: https://github.com/MCreator/MCreator/blob/e414c8c1a3e12009e2610205ce174d22c5b04ad3/plugins/mcreator-localization/help/default/plant/is_solid.md

Reply:
Entities collide even if your parameter is true but in a different way. This needs to be explained in this help tip